tests/test_run_inputs.py

```diff
@@ -0,0 +1,61 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+import time
+
+
+def test_run_input_header(driver: WebDriver):
+
+    driver.get("https://machinist.flapflap.io/")
```

Comment:
since for almost every test "https://machinist.flapflap.io/" is used maybe you can declare a global var

Reply:
What about running the tests on a local version of the app?

Technically, it should always be running on a local version rather than on the real app. At least if we could store this URL in a single variable somewhere, then we can change it easily for local testing